src/webview.ts

```diff
@@ -4,7 +4,10 @@ var Ansi = require('ansi-to-html');
 import "../assets/styles/webview.scss";
 import escapeHTML = require("escape-html");
 import * as paredit from "./cursor-doc/paredit";
-import { ModelEdit, ModelEditSelection } from "./cursor-doc/model";
+import { ModelEdit, ModelEditSelection, initScanner } from "./cursor-doc/model";
+
+const MAX_LINE_TOKENIZATION_LENGTH = 20000;
```

Comment:
Can we also use the vs code setting here? And do we want to?

Reply:
It would be nice to do that, but it wasn't immediately obvious to me how to do it.